modules/serverless-logic-persistence-db-schema-initialization.adoc

```diff
@@ -0,0 +1,87 @@
+
+When you are using `SonataFlow` with PostgreSQL persistence, you can initialize the database schema either by enabling Flyway or by manually applying database schema updates using DDL (Data Definition Language) scripts.
+
+Flyway is disabled by default. To enable Flyway, you configure it using one of the following methods:
+
+[id="serverless-logic-flyway-config-workflow-configmap_{context}"]
+== Flyway configuration in the workflow ConfigMap
+
+To enable Flyway in the workflow ConfigMap, you can add the following property:
+
+.Example of enabling Flyway in the workflow ConfigMap
+[source,yaml]
+----
+apiVersion: v1  
+kind: ConfigMap  
+metadata:  
+  labels:  
+    app: example-workflow  
+  name: example-workflow-props  
+data:  
+  application.properties: |  
+    quarkus.flyway.migrate-at-start = true
+----
+
+[id="serverless-logic-flyway-config-environment-variables_{context}"]
+== Flyway configuration using environment variables in the Workflow Container
+
+You can enable Flyway by adding an environment variable to the `spec.podTemplate.container` field in the `SonataFlow` CR bu using the following example:
+
+.Example of enabling Flyway by using the workflow container environment variable
+[source,yaml]
+----
+apiVersion: sonataflow.org/v1alpha08  
+kind: SonataFlow  
+metadata:  
+  name: example-workflow  
+  annotations:  
+    sonataflow.org/description: Example Workflow  
+    sonataflow.org/version: 0.0.1  
+spec:  
+  podTemplate:  
+    container:  
+      env:  
+        - name: QUARKUS_FLYWAY_MIGRATE_AT_START  
+          value: 'true'  
+  flow: ...
+----
+
+[id="serverless-logic-flyway-config-sonataflowplatform-properties_{context}"]
+== Flyway configuration using using SonataFlowPlatform properties
+
+To apply a common Flyway configuration to all workflows within a namespace, you can add the property to the `spec.properties.flow` field of the `SonataFlowPlatform` CR by shown in the following example:
+
+[NOTE]
+====
+This configuration is applied during workflow deployment. Ensure the Flyway property is set before deploying workflows.
+====
+
+.Example of enabling Flyway by using the `SonataFlowPlatform` properties
+[source,yaml]
+----
+apiVersion: sonataflow.org/v1alpha08  
+kind: SonataFlowPlatform  
+metadata:  
+  name: sonataflow-platform  
+spec:  
+  properties:  
+    flow:  
+      - name: quarkus.flyway.migrate-at-start  
+        value: true
+----
+
+[id="serverless-logic-manual-db-initialization-using-ddl_{context}"]
+== Manual database initialization using DDL scripts
+
+If you prefer manual initialization, you must disable Flyway by ensuring the `quarkus.flyway.migrate-at-start` property is either not configured or explicitly set to `false`.
+
+* By default, each workflow uses a schema name equal to the workflow name. Ensure that you manually apply the schema initialization for each workflow.
+
+* If you are using the `SonataFlow` CR persistence configuration, you can specify a custom schema name.
```

Comment:
```suggestion
* If you are using the `SonataFlow` CR persistence configuration, you can specify a custom schema name.

To execute the DDL scripts you must follow this procedure:

* Download the DDL scripts from the following location: link:{https://maven.repository.redhat.com/ga/org/kie/kogito/kogito-ddl/9.102.0.redhat-00005/kogito-ddl-9.102.0.redhat-00005-db-scripts.zip}[kogito-ddl-9.102.0.redhat-00005-db-scripts.zip]

* Extract the files

* Execute the .sql files present in the root directory on the target Postgresql database. You must execute the files following the given file version numbers.

For example:

. V1.35.0__create_runtime_PostgreSQL.sql
. V10.0.0__add_business_key_PostgreSQL.sql
. V10.0.1__alter_correlation_PostgreSQL.sql
. etc

[NOTE]
====
The file version numbers are not related with the {ServerlessLogicOperatorName} .
====

```

Reply:
@kaldesai Please check that this last adding is well formatted. I can't see the final layout.

For this link:     {https://maven.repository.redhat.com/ga/org/kie/kogito/kogito-ddl/9.102.0.redhat-00005/kogito-ddl-9.102.0.redhat-00005-db-scripts.zip}[kogito-ddl-9.102.0.redhat-00005-db-scripts.zip]

we need also a variable, since the link changes from version to version.
Many thanks!

Reply:
Added the content